setup.py

```diff
@@ -24,15 +24,15 @@
 from vsc.install.shared_setup import ag, jt
 
 PACKAGE = {
-    'version': '1.6',
+    'version': '1.6.1',
```

Comment:
why change the numbering scheme?

Reply:
to make it consistent again. @itkovian said he forgot the `.0` in his last PR.